content/ja/docs/reference/glossary/userns.md

```diff
@@ -0,0 +1,24 @@
+---
+title: User Namespace
+id: userns
+date: 2021-07-13
+full_link: https://man7.org/linux/man-pages/man7/user_namespaces.7.html
+short_description: >
+  非特権ユーザーに対して管理者権限をエミュレートするLinuxカーネルの機能。
+
+aka:
+tags:
+- security
+---
+
+root権限をエミュレートするLinuxカーネルの機能。rootlessコンテナを実現するために使われます。
+
+<!--more-->
+
+User Namespaceは、非rootユーザーが管理者(root)権限をエミュレートできるLinuxカーネルの機能です。例えば、コンテナの外部で管理者権限を持っていなくてもコンテナを実行することができます。
+
+User Namespaceは、コンテナブレークアウト攻撃による被害を軽減するのに効果的な対策です。
+
+ここでのUser Namespaceは、Linuxカーネルの機能を指し、Kubernetesの{{< glossary_tooltip text="Namespace" term_id="namespace" >}}とは異なります。
```

Comment:
I'd write “user namespace“ in lower case throughout. Does that feel right?

Reply:
I think either is fine as it's... a kind of proper noun right?
or I googled and it seems that some documentations like Docker, or some articles by gihyo or IT media, or some blogs, are even translating it as ユーザ(ー)名前空間.

cc @kubernetes/sig-docs-ja-reviews how do you feel, which one looks natural for you...?

Reply:
Use whatever is right for this localization, definitely.

I can state one thing: in _English_, _user namespace_ is not a proper noun.

Reply:
If it is a common noun, it would be better to translate it as `"ユーザー名前空間"` (in Japanese).
This is because the Japanese l10n guidelines state that common nouns should be translated, while proper nouns should not.

Given that "namespace" has been translated as "名前空間" in many pages, I think `"ユーザー名前空間"` is reasonable.

FYI: https://cs.k8s.io/?q=%E5%90%8D%E5%89%8D%E7%A9%BA%E9%96%93&i=nope&files=&excludeFiles=&repos=kubernetes/website

Reply:
OK, that sounds like a good option. I expect some readers will understand what ユーザー名前空間 means even without reading on.

Reply:
I've translated “user namespace“ to "ユーザー名前空間" as a common noun as @Okabe-Junya suggested.
Commit ID: 61b7bac51b9b329280e137b8cf160f6a4ba29421

@sftim @nasa9084 @Okabe-Junya 
Thank you for your detailed review!